scan/src/subscriptions/RedelegateSub.re

```diff
@@ -0,0 +1,90 @@
+type t = {balance: Coin.t};
+
+type validator_t = {
+  operatorAddress: Address.t,
+  moniker: string,
+  identity: string,
+};
+
+type redelegate_list_t = {
+  amount: Coin.t,
+  completionTime: MomentRe.Moment.t,
+  validator: validator_t,
+};
+
+module RedelegationByDelegatorConfig = [%graphql
+  {|
+    subscription UnbondingByDelegator($limit: Int!, $offset: Int!, $delegator_address: String!, $current_time: timestamp) {
+      accounts_by_pk(address: $delegator_address) {
+        redelegations(offset: $offset, limit: $limit, order_by: {completion_time: asc}, where: {completion_time: {_gte: $current_time}}) @bsRecord{
+          amount @bsDecoder(fn: "GraphQLParser.coin")
+          completionTime: completion_time @bsDecoder(fn: "GraphQLParser.timestamp")
+          validator @bsRecord{
+            operatorAddress: operator_address @bsDecoder(fn: "Address.fromBech32")
+            moniker
+            identity
+          }
+        }
+      }
+    }
+  |}
+];
```

Comment:
In redelegation, we'd like to display `sourceValidator`, `destinationValidator`.